examples/ImageRecon/GEOmetrics/README.md

```diff
@@ -1,31 +1,38 @@
 # Example: GeoMetrics
-This example allows you to train and test the GEOMetrice algorithm for signle image 3D object reconstruction. For details on the algorithms see: "GEOMetrics: Exploiting Geometric Structure for Graph-Encoded Objects" : https://arxiv.org/abs/1901.11461
+This example allows you to train and test the GEOMetrics algorithm for 
+single image 3D object reconstruction. For details on the algorithms see: 
+"GEOMetrics: Exploiting Geometric Structure for Graph-Encoded Objects": 
+https://arxiv.org/abs/1901.11461
 
 
 ### Training the network:
 
-To train the system call
-```
-python train.py
+#### Option 1 : No latent embedding loss (faster to train)
+```bash
+python train.py --shapenet-root <path/to/ShapeNet> \
+    --shapenet-rendering-root <path/to/ShapeNetRendering> \
+    --cache-dir cache/
```

Comment:
Can we add detail on the purpose of `--cache-dir` and the duration of training ? (maybe with the step of the preprocessing)

Reply:
Done.